source/gameengine/Ketsji/KX_KetsjiEngine.cpp

```diff
@@ -1744,34 +1745,27 @@ bool KX_KetsjiEngine::GetShowArmatures() const
 	return m_showArmature;
 }
 
-void KX_KetsjiEngine::DrawCameraFrustum(KX_Camera *cam, KX_Scene *scene)
+void KX_KetsjiEngine::DrawCameraFrustum(KX_Camera *cam, KX_Scene *scene, const RAS_Rect& viewport)
 {
 	bool perspective = cam->GetCameraData()->m_perspective;
 
-	Scene *sc = scene->GetBlenderScene();
-	float aspx = (float)sc->r.xsch * sc->r.xasp;
-	float aspy = (float)sc->r.ysch * sc->r.yasp;
-	float ratiox = min_ff(aspx / aspy, 1.0f);
-	float ratioy = min_ff(aspy / aspx, 1.0f);
```

Comment:
@panzergame : This code was intended to match this behaviour: https://www.youtube.com/watch?v=be-ku3YVgZ8 beginning of the video